.github/workflows/build.yml

```
@@ -0,0 +1,44 @@
+name: Build and push Docker images
+
+on:
+  push:
+    branches:
+      - master
+      - build*
+  pull_request:
+    branches:
+      - master
+      - build*
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.CR_PAT }}
+      -
+        name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
```

Vi skulle kunna skriva detta så här:
```suggestion
          push: ${{ github.event_name != 'pull_request' }}
```

Då pushar vi endast nya images efter en PR är mergad :)

---

Bra att veta! Första gången jag leker med GHA. I det här fallet är det dock tanken att vi skall pusha nya images vid varje bygge, så att när du vill testa något kan du låta Github bygga dem istället för att behöva göra det på din egen maskin. :)